src/views/set2/Set2.vue

```diff
@@ -2096,57 +2103,63 @@
+
+          // Select the associated temperature line
+          self.d3.selectAll(".temp_line.seg_" + segment_id)
+            .style("stroke", "#fcba03")
+            .style("stroke-width", 3)
+            .raise()
+          // // select all spatial rectangles and make mostly opaque to dim matrix
+          // self.d3.selectAll(".c2p3.matrixSpatialRect")
+          //     .style("opacity", 0.6)
+          //     .style("stroke-width", 2);
+          // // select background rectangle and change filter
+          // self.d3.selectAll(".c2p3.matrixBkgdRect")
+          //     .attr("filter", "url(#shadow3)")
+          // // select all *temporal* rectangles and repurpose for bigger view
+          // if (self.segmentDict[segment_id].year_count['2019'] > 0) {
+          //     for (let i = 0; i < self.myGroups_c2p3.length; i++) {
+          //         let seg_day = self.myGroups_c2p3[i]
+          //         // for all segment-days with observations...
+          //         if (self.segmentDict[segment_id]['data_2019_daily'][seg_day] != undefined) {
+          //             if (self.segmentDict[segment_id]['data_2019_daily'][seg_day]['day_count'] > 0) {
+          //                 // repurpose the temporal rectangles to show the temperatures over the year
+          //                 self.d3.selectAll(".c2p3.matrixTemporalRect.time" + seg_day)
+          //                     .attr("height", cellHeight_c2p3)
+          //                     .attr("y", function(d) {
+          //                       return yScale_matrix_c2p3(segment_id) - cellHeight_c2p3/2;
+          //                     })
+          //                     .style("fill", function(d) {
+          //                         return self.tempColor(self.segmentDict[segment_id]['data_2019_daily'][seg_day]['day_t_c'])
+          //                     })
+          //                     .style("stroke", function(d) {
+          //                         return self.tempColor(self.segmentDict[segment_id]['data_2019_daily'][seg_day]['day_t_c'])
+          //                     })
+          //                     .style("opacity", 1)
+          //                     .raise()
+          //             }
+          //         }
+          //     }
+          // }
+          // // select the spatial rectangle corresponding to the highlighted segment
+          // // set stroke width, opacity, and stroke color
+          // // based on whether segment has any observations in record
+          // if (self.segmentDict[segment_id].year_count['2019'] == 0) {
+          //     // select the spatial rectangle corresponding to the highlighted segment
+          //     self.d3.selectAll(".c2p3.matrixSpatialRect.seg" + segment_id)
+          //       .style("stroke-width", 0.5)
+          //       .style("opacity", 1)
+          //       .style("stroke", "#e0e0e0")
+          //       .attr("height", 3)
+          //       // raise the spatial rectangle
+          //       .raise()
```

Comment:
👀

Reply:
😉

Reply:
😂 so much commented-out code 😬. I promise I'll clean this all up once we merge together this PR, upstream, and the data 2020 PR!